### PR TITLE
add support for all redis scaler list types

### PR DESCRIPTION
**Pull request by ilyamor**
Relates to https://github.com/kedacore/keda/pull/1006

### CODE REVIEW

File: content/docs/2.0/scalers/redis-lists.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ go_file = "redis_scaler"
 
 ### Trigger Specification
```
Comment (tomkerkhove) [inline content/docs/2.0/scalers/redis-lists.md:10]: Can you update the scaler title and description please?

File: content/docs/2.0/scalers/redis-lists.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ go_file = "redis_scaler"
...
 
-This specification describes the `redis` trigger that scales based on the length of a list in Redis.
+This specification describes the `redis` trigger that scales based on the length of a list,hash,set or sorted-set in Redis.
```
Comment (tomkerkhove) [inline content/docs/2.0/scalers/redis-lists.md:12]: Can you please make it a bit more readible please?
```suggestion
This specification describes the `redis` trigger that scales based on the length of a list, hash, set or sorted-set in Redis.
```

File: content/docs/2.0/scalers/redis-lists.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ go_file = "redis_scaler"
...
 
 ```yaml
```
Comment (tomkerkhove) [inline content/docs/2.0/scalers/redis-lists.md:14]: Not sure if `listName` & `listLength` are the ideal names for this

Comment (zroubalik) [reply content/docs/2.0/scalers/redis-lists.md:14]: IMHO we can keep the old ones, If we don't come up with better names. (I don't have a better idea 🤷‍♂️ 😄 )

**Review by zroubalik — CHANGES_REQUESTED** [review]
Please fix the formatting and correct DCO (commit is not signed).  Other LGTM

**Comment by zroubalik** [comment]
@ilyamor do you think you can update this PR? Thanks!

**Comment by tomkerkhove** [comment]
Any update on this @ilyamor?

**Comment by tomkerkhove** [comment]
Any update on this @ilyamor?

**Comment by tomkerkhove** [comment]
Closing due to no response

**Pull request closed without merging**